collections/brandedUserProfile.js

```diff
@@ -332,3 +333,9 @@ BrandedUserSchema = new SimpleSchema({
 
 BrandedUserProfiles = new Mongo.Collection('brandedUserProfiles')
 BrandedUserProfiles.attachSchema(BrandedUserSchema)
+
+BrandedUserProfiles.before.update((userId, doc, fieldNames, modifier) => {
+  if (modifier.$set && modifier.$set['_contactSettings.canText'] !== undefined) {
+    modifier.$set['_contactSettings.canTextLastModified'] = new Date(Date.now())
```

Comment:
This is more just a curiousity, but doesn't new date create the current date? Is Date.now() needed as an argument, maybe there is some caviat I'm unaware of here?

Reply:
I had this as just Date.now() before, but it was throwing an error. I *believe* it is because Date.now() returns a number, but Mongo requires that the value of a `Date` type field be a `Date` object. But not totally clear.

Reply:
You'r right, I got confused. I'll change this.